src/OneDriveGUI.py

```diff
@@ -1905,11 +1904,11 @@ def save_global_config():
 
     main_window = MainWindow()
     settings_window = SettingsWindow()
-    settings_window.hide()
 
     if gui_settings["SETTINGS"]["start_minimized"] == "True":
         main_window.hide()
     else:
         main_window.show()
+        settings_window.show()
```

Comment:
Thanks for the contribution @andreztz . Nice catch there of this old leftover. It seems XFCE could not really handle the very quick unhide and hide window change :-).

Just please remove this line 1912 as it would cause the setting windows to always show on startup for users who do not have the start_minimized feature enabled. None of the lines 554, 1908 and 1912 are actually needed. I will then merge it.